test/CodeUnits.purs

```diff
@@ -71,36 +75,47 @@ testCodeUnits = do
 
   assert $ expectResult 3 nested "(((a)))"
-  assert $ expectResult ("a":"a":"a":Nil)  (many (string "a")) "aaa"
+  assert $ expectResult ("a" : "a" : "a" : Nil) (many (string "a")) "aaa"
   assert $ parseFail (many1 (string "a")) ""
-  assert $ canParse (parens (do
-    _ <- string "a"
-    optionMaybe $ string "b")) "(ab)"
-  assert $ expectResult (NonEmptyList ("a" :| "a":"a":Nil)) (string "a" `sepBy1` string ",") "a,a,a"
-  assert $ canParse (do
-    as <- string "a" `endBy1` string ","
-    eof
-    pure as) "a,a,a,"
+  assert $ canParse
+    ( parens
+        ( do
+            _ <- string "a"
+            optionMaybe $ string "b"
+        )
+    )
+    "(ab)"
+  assert $ expectResult (NonEmptyList ("a" :| "a" : "a" : Nil)) (string "a" `sepBy1` string ",") "a,a,a"
+  assert $ canParse
+    ( do
+        as <- string "a" `endBy1` string ","
+        eof
+        pure as
+    )
+    "a,a,a,"
   assert' "opTest" $ expectResult "abc" opTest "a+b+c"
   assert' "exprTest" $ expectResult (-3) exprTest "1*2+3/4-5"
-  assert' "tryTest "$ canParse tryTest "aacc"
-  assert $ expectResult (NonEmptyList ('0' :| '1':'2':'3':'4':Nil)) (many1 anyDigit) "01234/"
-  assert $ expectResult (NonEmptyList ('5' :| '6':'7':'8':'9':Nil)) (many1 anyDigit) "56789:"
+  assert' "tryTest " $ canParse tryTest "aacc"
+  assert $ expectResult (NonEmptyList ('0' :| '1' : '2' : '3' : '4' : Nil)) (many1 anyDigit) "01234/"
+  assert $ expectResult (NonEmptyList ('5' :| '6' : '7' : '8' : '9' : Nil)) (many1 anyDigit) "56789:"
   assert $ expectResult "aaaa" (regex "a+") "aaaab"
-  assert $ expectResult ("a":"a":"a":Nil)  (manyTill (string "a") (string "b")) "aaab"
+  assert $ expectResult ("a" : "a" : "a" : Nil) (manyTill (string "a") (string "b")) "aaab"
   assert $ expectResult Nil (manyTill (string "a") (string "b")) "b"
-  assert $ expectResult (NonEmptyList ("a" :| "a":"a":Nil)) (many1Till (string "a") (string "b")) "aaab"
+  assert $ expectResult (NonEmptyList ("a" :| "a" : "a" : Nil)) (many1Till (string "a") (string "b")) "aaab"
   assert $ parseFail (many1Till (string "a") (string "b")) "b"
   -- check correct order
-  assert $ expectResult (NonEmptyList ('a' :| 'b':'c':Nil)) (many1Till anyChar (string "d")) "abcd"
+  assert $ expectResult (NonEmptyList ('a' :| 'b' : 'c' : Nil)) (many1Till anyChar (string "d")) "abcd"
 
   log "Running overflow tests (may take a while)"
 
   -- check against overflow
   assert $ canParse (many1Till (string "a") (string "and")) $ (fold <<< take 10000 $ repeat "a") <> "and"
 
   assert' "many should not blow the stack" $ canParse (many (string "a")) (SC.joinWith "" $ replicate 100000 "a")
-  assert' "many failing after" $ parseFail (do
-    as <- many (string "a")
-    eof
-    pure as) (SC.joinWith "" (replicate 100000 "a") <> "b" )
\ No newline at end of file
+  assert' "many failing after" $ parseFail
+    ( do
+        as <- many (string "a")
+        eof
+        pure as
+    )
+    (SC.joinWith "" (replicate 100000 "a") <> "b")
```

Comment:
Do we care about newlines being added at the end of the file?

Reply:
Hm, yes — I thought purs-tidy automatically ensured a single trailing newline (cc @natefaubion) but perhaps not.